modules/appeals_api/spec/controllers/concerns/appeals_api/character_handling_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+class FakeController < ApplicationController
+  include AppealsApi::CharacterHandling
+end
+
+describe FakeController do
+  let(:higher_level_review) { create(:extra_higher_level_review_v2) }
+
+  describe '#transliterate_name' do
```

Comment:
Needs more specs to test all the things the method  does, like stripping whitespace and truncating to 50 characters (both can be tested at once if you want).

Reply:
This is going away because your bug ticket accomplishes all the things.